resources/elasticbeanstalk_applications.go

```diff
@@ -0,0 +1,131 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func ElasticbeanstalkApplications() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_elasticbeanstalk_applications",
+		Description:  "Describes the properties of an application.",
+		Resolver:     fetchElasticbeanstalkApplications,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn", "date_created"}},
```

Comment:
why date created? arn isn't unique enough ? I don't think its possible for this collision to occur.

Reply:
I was thinking for history support this would enable us to differentiate between a resource that was deleted then recreated with a new name

Reply:
you don't need to do that, history automatically adds fetch date to each PK.